lib/iiif_manifest/v3/manifest_builder/thumbnail_builder.rb

```diff
@@ -0,0 +1,52 @@
+module IIIFManifest
+  module V3
+    class ManifestBuilder
+      class ThumbnailBuilder
+        attr_reader :display_content, :iiif_thumbnail_factory, :image_service_builder_factory
+        def initialize(display_content, iiif_thumbnail_factory:, image_service_builder_factory:)
+          @display_content = display_content
+          @iiif_thumbnail_factory = iiif_thumbnail_factory
+          @image_service_builder_factory = image_service_builder_factory
+        end
+
+        # @param canvas_or_manifest [#thumbnail=]
+        def apply(canvas_or_manifest)
+          build_thumbnail
+          image_service_builder.apply(thumbnail)
+          canvas_or_manifest.thumbnail = thumbnail
+        end
+
+        private
+
+        def build_thumbnail
+          thumbnail['id'] = File.join(display_content.iiif_endpoint.url, 'full', '!200,200', '0', 'default.jpg')
+          thumbnail['height'] = (display_content.height * reduction_ratio).round
+          thumbnail['width'] = (display_content.width * reduction_ratio).round
+          thumbnail['format'] = display_content.format
+        end
+
+        def reduction_ratio
+          width = display_content.width
+          height = display_content.height
+          max_edge = 200.0
```

Comment:
Should this be configurable?  We have a config file now.

Reply:
I agree, perhaps another ticket or should it be handled in this one?

Reply:
I agree that this should be configurable.  I think it would be good to handle it in this PR if possible.  Maybe `max_edge` should be it's own private method since I think it should be used to make the image api url on line 22 as well.